component/async/kafka/group/group.go

```diff
@@ -114,7 +114,7 @@ func (c *consumer) Consume(ctx context.Context) (<-chan async.Message, <-chan er
 		return nil, nil, fmt.Errorf("failed to create consumer: %w", err)
 	}
 	c.cg = cg
-	log.Infof("consuming messages from topics '%s' using group '%s'", strings.Join(c.topics, ","), c.group)
+	log.Debugf("consuming messages from topics '%s' using group '%s'", strings.Join(c.topics, ","), c.group)
```

Comment:
consider keeping this, as it confirms the working of the component. It s very useful in debugging issues in prod.

Reply:
We have prometheus metrics for this, would that be enough?

Reply:
ok, sounds good as well.